swagger2/src/main/resources/com/cerner/beadledom/swagger2/ui.mustache

```diff
@@ -0,0 +1,52 @@
+<!-- HTML for static distribution bundle build -->
```

Comment:
Did you create this file, or was it copied from somewhere? I just am wondering if we need to track on changes elsewhere.

Reply:
It's based on the starter index.html file from swagger-ui but I changed the javascript properties to simplify it and use `BaseLayout` as well as replacing the url with a template variable.